src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/webdriver/ElementUtils.kt

```diff
@@ -0,0 +1,43 @@
+package com.atlassian.performance.tools.jiraactions.api.webdriver
+
+import com.atlassian.performance.tools.jiraactions.api.page.wait
+import org.openqa.selenium.By
+import org.openqa.selenium.Keys
+import org.openqa.selenium.WebDriver
+import org.openqa.selenium.WebElement
+import org.openqa.selenium.support.ui.ExpectedConditions
+
+class ElementUtils {
```

Comment:
```suggestion
class HardenedKeyboard {
```

- [ ] consider a less vague name

Reply:
If we'll harden it more it'll become `EvenMoreHardenedKeyboard` :)